make.sh

```diff
@@ -8,7 +8,15 @@ opam_setup() {
   set -x
   opam init -y -a --bare $SANDBOXING # sandboxing is disabled in travis and docker
   opam update
-  opam switch -y create . --deps-only ocaml-base-compiler.4.14.0 --locked
+  opam switch -y create . --deps-only ocaml-base-compiler.4.14.0 --update-invariant
```

Comment:
The lack of `--locked` due to the need for `--update-invariant` ignores the lock file and thus breaks precise reproducibility. That's why it's better to just not offer two variants, but only have one which uses the lock file and doesn't require update-invariant.

Reply:
Should be fixed in https://github.com/goblint/analyzer/pull/928/commits/7013805acce83979b3eea329d454dcb82ff1e158